src/components/App/App.component.jsx

```diff
@@ -1,56 +1,21 @@
-import React, { useLayoutEffect } from 'react';
+import React from 'react';
 import { BrowserRouter, Switch, Route } from 'react-router-dom';
 
-import AuthProvider from '../../providers/Auth';
-import HomePage from '../../pages/Home';
-import LoginPage from '../../pages/Login';
-import NotFound from '../../pages/NotFound';
-import SecretPage from '../../pages/Secret';
-import Private from '../Private';
-import Fortune from '../Fortune';
-import Layout from '../Layout';
-import { random } from '../../utils/fns';
+import HomePage from '../../pages/Home/Home.page';
+import Layout from '../atoms/Layout';
+import Header from '../Organisms/Header/index';
```

Comment:
Try to be consistent with the name casing, e.g. should be `'../organisms/Header/index'` with lowercase `o` like in `'../atoms/Layout'`?

Reply:
Renamed the atoms folder to Uppercase